bluebrain/repo-patches/packages/py-pyarrow/package.py

```diff
@@ -5,10 +5,23 @@
 class PyPyarrow(BuiltinPyPyarrow):
     __doc__ = BuiltinPyPyarrow.__doc__
 
+    pypi = 'pyarrow/pyarrow-9.0.0.tar.gz'
+
+    version('9.0.0', sha256='7fb02bebc13ab55573d1ae9bb5002a6d20ba767bf8569b52fce5301d42495ab7')
```

Comment:
FYI, with pyarrow 9.0.0 we had issues in the CI, when the CPU id 0 is not available. See https://issues.apache.org/jira/browse/ARROW-17319
cc @mgeplf

Reply:
Hm, good point. Did you observe that with 8.0.0? I could just mark that as `preferred` for the time being.

Reply:
@matz-e how does pyarrow pull in it's `aws-c-*` dependencies?

Reply:
8 wheels seemed to be fine, but I don't think we tracked when they started using the `aws` stuff.

Reply:
@mgeplf it does not pull in any dependencies on AWS, we also build without S3 support. So I guess this should be fine? I'll try some reproducer on BB5

Reply:
Hm, don't see this with py3.9 using a virtualenv with a `pyarrow` wheel, either.

Reply:
OK, so could confirm the issue with a PYPI provided `pyarrow`. As we don't build with any AWS components, `py-pyarrow@9.0.0` is fine on BlueBrain5, though.